mfhdf/hrepack/hrepacktst.c

```diff
@@ -1450,7 +822,7 @@ add_sd(const char  *fname,       /* file name */
     }
     /* initialize dimension scales */
     for (i = 0; i < X_DIM; i++)
-        data_X[i] = i;
+        data_X[i] = (int16)i;
```

Comment:
A lot of data assignment warnings like this. Since this is just test code, I just cast them away.